compiler/rustc_span/src/lib.rs

```diff
@@ -1765,6 +1768,7 @@ impl<S: SpanEncoder> Encodable<S> for SourceFile {
         self.checksum_hash.encode(s);
         // Do not encode `start_pos` as it's global state for this session.
         self.source_len.encode(s);
+        self.original_source_len.encode(s);
```

Comment:
Not sure if this is really needed

Reply:
I would say if the normalized source len is encoded, then unnormalized source len should also be encoded. I would imagine that if the source file changed, then any persisted incr cache should be invalidated.